### PR TITLE
over_flow_fix

### PR DESCRIPTION
**Pull request by Tushar403**
Targets #1716 

### Description
add members & fixed overflow in the popup screen 

### Checklist:
 [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php [hunk 1]
```diff
@@ -58,6 +58,72 @@ class="reader_image"
...
+                                                    class="reader_image"
+                                                    src="{{ $reader->avatar}}"
+                                                    alt="{{ $reader->name }}"/>
+                                            </div>
+                                            <div class="text-center mr-2">
+                                                <img
+                                                    data-toggle="tooltip"
+                                                    title="{{ $reader->name }}"
+                                                    data-placement="bottom"
+                                                    class="reader_image"
```
Comment (GauravGusain98) [inline resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php:84]: Why you are repeating the code so this many times. For a user the image should appear only one time. According your current implementation, the logic will print the image of a user 8 times as in every loop iteration 8 div will be printed.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php (hunk 1 above)
Comment (Tushar403) [reply resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php]: corrected my mistakes please check and review it, and let me if any changes are needed.

**Review by kuldeep3 — COMMENTED** [review]
As this PR contains UI changes, could you provide a screenshot of how it was before and how it is looking after these changes? @Tushar403

**Comment by Tushar403** [comment]
> As this PR contains UI changes, could you provide a screenshot of how it was before and how it is looking after these changes? @Tushar403
Screenshot (20)
(before changes)
Screenshot (21)
(after changes)

**Review by kuldeep3 — APPROVED** [review]

**Comment by Tushar403** [comment]
@kuldeep3  so I am done with my task. Is it ready to merge?

**Comment by P4NK4J** [comment]
@GauravGusain98 You need to submit your review here as well. Please take a look.

**Pull request merged**